src/pages/fr/install/auto.mdx

```diff
@@ -91,14 +87,39 @@ Astro va également suivre les modifications de fichiers dans le répertoire `sr
 
 Si vous n'arrivez pas à ouvrir votre projet dans le navigateur, revenez au terminal où vous avez lancé le script `dev` pour voir ce qui a mal tourné, ou si votre projet est servi sur une url différente que celle mentionnée ci-dessus.
 
+Vous pouvez également démarrer un nouveau projet Astro basé sur un dépôt GitHub existant en passant un argument `--template` à la commande `create astro`.
+
+## Modèles de démarrage
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm create astro@latest -- --template <nom-d-utilisateur-github>/<depot-github>
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm create astro@latest --template <nom-d-utilisateur-github>/<depot-github>
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn create astro --template <nom-d-utilisateur-github>/<depot-github>
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+Explorez notre [galerie de thèmes et de modèles de démarrage](https://astro.build/themes/) où vous pouvez parcourir des thèmes pour des blogs, des portfolios, de la documentation, des pages d'atterrissage, et plus encore! Ou, [recherchez sur GitHub](https://github.com/search?o=desc&q=astro+starter&s=stars&type=Repositories) pour encore plus de projets de démarrage.
```

Comment:
To be seen, but personally, I think it's better to leave the term `landing page` rather than `pages d'atterrissage` which is not used in French (or not to my knowledge). I looked at the proposed translations on [Deepl](https://www.deepl.com/) to make sure there were no other possible translations. Here are the proposed ones: [https://www.deepl.com/translator#en/fr/landing page](https://www.deepl.com/translator#en/fr/landing%20page)
```suggestion
Explorez notre [galerie de thèmes et de modèles de démarrage](https://astro.build/themes/) où vous pouvez parcourir des thèmes pour des blogs, des portfolios, de la documentation, des landing page, et plus encore ! Ou, [recherchez sur GitHub](https://github.com/search?o=desc&q=astro+starter&s=stars&type=Repositories) pour encore plus de projets de démarrage.
```

Reply:
Thanks @pioupia for spotting this, this is indeed invalid